kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -155,7 +156,14 @@ export class UnityInterface {
     this.SendMessageToUnity('Main', 'DumpRendererLockersInfo')
   }
 
-  public CreateUIScene(data: { id: string; baseUrl: string }) {
+  public CreateUIScene(data: {
```

Comment:
can we rename this to `CreateGlobalScene` or something like that?

Reply:
Yes, you're totally right 👍 Done! Thanks!